<ui version="4.0" >
 <class>SolutionVerifForm</class>
 <widget class="QWidget" name="SolutionVerifForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Mesh quality criteria</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>17</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Post-processing format</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxFMTCHR" >
            <property name="minimumSize" >
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxOptions" >
        <property name="title" >
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <layout class="QHBoxLayout" >
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="labelformat" >
               <property name="text" >
                <string>format</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QComboBox" name="comboBoxFormat" >
               <property name="minimumSize" >
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QLabel" name="labelPolygon" >
               <property name="text" >
                <string>polygons</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="QComboBox" name="comboBoxPolygon" >
               <property name="minimumSize" >
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip" >
                <string>Code_Saturne key word: OPTCHR</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" >
              <widget class="QLabel" name="labelPolyhedra" >
               <property name="text" >
                <string>polyhedra</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" >
              <widget class="QComboBox" name="comboBoxPolyhedra" >
               <property name="minimumSize" >
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip" >
                <string>Code_Saturne key word: OPTCHR</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" >
              <widget class="QLabel" name="labelBigEndian" >
               <property name="toolTip" >
                <string>Code_Saturne key word: OPTCHR</string>
               </property>
               <property name="text" >
                <string>big-endian</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1" >
              <widget class="QCheckBox" name="checkBoxBigEndian" >
               <property name="toolTip" >
                <string>Code_Saturne key word: OPTCHR</string>
               </property>
               <property name="text" >
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>17</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonBatch" >
          <property name="text" >
           <string>Check mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>17</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
